controllers/managedocs_controller_test.go

```diff
@@ -494,6 +506,18 @@ var _ = Describe("ManagedOCS controller", func() {
 		}
 	}
 
+	assertPrometheusTLSConfigValues := func(ctx context.Context,
+		client client.Client) {
+		prom := promTemplate.DeepCopy()
+		Expect(client.Get(ctx, utils.GetResourceKey(prom), prom)).Should(Succeed())
+		Expect(prom.Spec.Web.TLSConfig.KeySecret.LocalObjectReference.Name == prometheusServiceSecretName).Should(BeTrue())
+		Expect(prom.Spec.Web.TLSConfig.Cert.Secret.LocalObjectReference.Name == prometheusServiceSecretName).Should((BeTrue()))
+		Expect(prom.Spec.Web.TLSConfig.ClientCA.ConfigMap.LocalObjectReference.Name == servingCertCABundleConfigMapName).Should(BeTrue())
+		Expect(prom.Spec.Web.TLSConfig.ClientCA.ConfigMap.Key == "service-ca.crt").Should(BeTrue())
+		Expect(prom.Spec.Web.TLSConfig.KeySecret.Key == "tls.key").Should(BeTrue())
+		Expect(prom.Spec.Web.TLSConfig.Cert.Secret.Key == "tls.crt").Should(BeTrue())
```

Comment:
Please use `.Should(Equal(<value>))` instead of `==` and `Should(BeTrue())

For example:
```
Expect(prom.Spec.Web.TLSConfig.KeySecret.LocalObjectReference.Name).Should(Equal(prometheusServiceSecretName))
```

Reply:
Done